qcfractal/cli/qcfractal_manager.py

```diff
@@ -440,6 +442,14 @@ class ParslProviderSettings(SettingsBlocker):
 
     """
 
+    def __init__(self, **kwargs):
+        if 'nodes_per_block' in kwargs:
+            raise ValueError('``nodes_per_block`` is named ``nodes_per_job`` to keep nomenclature consistent')
```

Comment:
We may want to keep kwargs the same us the underlying adapter. A bit on the fence about this.

Reply:
I was on the fence as well and would be happy with "block" or "job".

What sent me over to "nodes_per_job" is that the documentation for the different adapters took awhile to understand and required frequent consulting with the nomenclature definitions. So, I did not want to further introduce confusion by adding new terms unnecessarily.

Reply:
We don't see to use the `job` notion that often. Perhaps leaving it as `nodes_per_block` with an explanation may be best. Something like "`nodes_per_block` allows a single job to span multiple nodes with a worker on each node, this is useful to limit the number of jobs submitted to the resource scheduler". Though that highlights the block/job issue, but I do like attempting to keep the scheduler specific kwargs equal to their own specifications.

We could also consider adding ``nodes_per_job`` to the common settings and enforce everyone besides Parsl being one.

@mattwelborn

Reply:
I like that idea of it being a common setting best, as notions of requesting batches of nodes in a single job is not a Parsl-specific thing. Is it more of a "cluster" common setting though?

Reply:
My reasoning for the `common` setting is this is where the amount of resources requested is specified. `cluster` seems a bit more ancillary about the "how".

Reply:
I should note that we should probably shake up the names and organization once more before 1.0. The shake up is not deep since most seem fine, but worth another reconsideration.

We should also support building one of these in the python layer as a first class citizen. This likely requires interrogation of the queue adapter objects for number of cores/memory that they have set internally.